packages/node_modules/@webex/internal-plugin-calendar/src/index.js

```diff
@@ -65,22 +65,40 @@ registerInternalPlugin('calendar', Calendar, {
             return Promise.resolve();
           }
 
-          // Decrypt participant properties
-          const decryptedParticipants = object.encryptedParticipants.map((participant) => Promise.all([
-            ctx.transform('decryptTextProp', 'encryptedEmailAddress', object.encryptionKeyUrl, participant),
-            ctx.transform('decryptTextProp', 'encryptedName', object.encryptionKeyUrl, participant)
-          ]));
+          // Check if notes and participants were sent with meeting object, and fetch if not
+          if (!object.encryptedNotes) {
+            object.encryptedNotes = Calendar.prototype.getNotes(object.id);
+          }
+          if (!object.encryptedParticipants) {
+            object.encryptedParticipants = Calendar.prototype.getParticipants(object.id);
+          }
+
+          return Promise.all([object.encryptedNotes, object.encryptedParticipants])
```

Comment:
Is it possible for either of these promises to reject? No change is needed if they cannot be rejected, otherwise, we should discuss what needs to occur in case of rejection.

Reply:
Only scenario I can think of is if a network error occurs and the Promise stays pending. Otherwise, since we're using a meeting id that we just received, these shouldn't reject.